doc/news/changes/incompatibilities/20221205Schreter

```diff
@@ -0,0 +1,5 @@
+Removed: A subcell with equal iso-level values at the vertices, larger than the cut 
+iso level, is no longer considered as a cut cell. The special case was removed from the 
+1D implementation of GridTools::MarchingCubeAlgorithm.
+<br>
+(Magdalena Schreter, 2022/12/05)
```

Comment:
```suggestion
```
Not needed, since the 1D implementation was only added since the last release and the change-log entries are only relevant during the next release. If you want, you can still update the old change-log entry.